keyboards/bastardkb/dilemma/3x5_2/keymaps/pegak/features/autocorrection.c

```diff
@@ -0,0 +1,204 @@
+// Copyright 2021-2022 Google LLC
```

Comment:
Autocorrect is now a core QMK feature and should be used over this — [https://docs.qmk.fm/#/feature_autocorrect](https://docs.qmk.fm/#/feature_autocorrect)